Code/Support/RestKit-Prefix.pch

```diff
@@ -5,6 +5,11 @@
 #ifdef __OBJC__
     #import <Foundation/Foundation.h>
 
+// Fix for compilation on iOS 6 where this is not defined.
+#if !defined NS_DESIGNATED_INITIALIZER
```

Comment:
Samuel,

Go ahead and change it yourself.

I've patched the issue as it is a bug with the library and prevents it compiling on older platforms and if you're going to be petty enough to want a different version of the fix because you prefer the way it looks, then you can do it yourself. If your objection to the fix had any kind of technical justification, we could have a discussion, but refusing to merge a fix because you don't like the formatting is astounding to me.

People like you, Samuel, detract from the spirit of open source and really deter people from wanting to help out on projects. I've seen it with other open source projects and as a result I've stopped using them and stopped contributing. 




> On Jun 25, 2015, at 10:55 PM, Samuel E. Giddins <notifications@github.com> wrote:
> 
> In Code/Support/RestKit-Prefix.pch:
> 
> > @@ -5,6 +5,11 @@
> >  #ifdef __OBJC__
> >      #import <Foundation/Foundation.h>
> >  
> > +// Fix for compilation on iOS 6 where this is not defined.
> > +#if !defined NS_DESIGNATED_INITIALIZER
> yes, please do change it
> 
> —
> Reply to this email directly or view it on GitHub.
>